android-components/components/browser/storage-sync/src/main/java/mozilla/components/browser/storage/sync/PlacesHistoryStorage.kt

```diff
@@ -247,6 +250,27 @@ open class PlacesHistoryStorage(
         }
     }
 
+    /**
+     * Enqueues a periodic storage maintenance worker to WorkManager that prunes database entries
+     * when it exceeds [PlacesHistoryStorageWorker.DB_SIZE_LIMIT_IN_BYTES].
+     */
+    override fun registerStorageMaintenanceWorker() {
+        WorkManager.getInstance(context).enqueueUniquePeriodicWork(
+            PlacesHistoryStorageWorker.UNIQUE_NAME,
```

Comment:
Self-note: we're using the same `PlacesHistoryStorageWorker.UNIQUE_NAME` here and for the work request tag. This is okay because we only have one unique work request so using the same tag has no unintended side-effects if we were to query for work by either.

Reply:
Yes, i had exactly the same note to myself while coding :)